### ISSUE TITLE
Make Quiet downloadable/updateable via package managers

### ISSUE THREAD

**Issue by richardtorres314** [TryQuiet/quiet#2247]
It would be neat to see Quiet available to package managers such as:

* Homebrew on Mac & Linux
* WinGet on Windows

Not sure if there are other package managers that would make sense to be available on, but those could be added to the list.

**Comment by holmesworcester** [comment]
I would love there to be packages for the major Linux distros. We ship a Mac build so I'm curious what the use case is for homebrew, but having packages for Debian-based distros would be awesome!

**Comment by richardtorres314** [comment]
Same reason as for the Linux distros. It makes installing, updating, deleting, etc apps uniform and accessible via CLI. There are probably other CLI package managers for Mac but Homebrew is certainly one of the most, if not the most, widely used one. +1 on the Debian-based distros too!

**Comment by holmesworcester** [comment]
Okay, I understand! Well yeah, go for it! There might be higher priority things but if you're into doing this I support that!

**Comment by richardtorres314** [comment]
Totally get that! I'm currently working on another issue, but if I have time I'll come back to this. Thank you!

**Comment by richardtorres314** [comment]
@holmesworcester I've added the app to Homebrew (https://formulae.brew.sh/cask/quiet). Looks like it's already gotten some downloads through that! Thoughts on which package manager(s) on Debian-based distros? I'll also take a look into at least Winget on the Windows side of things. Thank you!

**Comment by holmesworcester** [comment]
Awesome! What do you think is best re: package managers for Debian distros?

Apt seems like the place to start, no?

**Comment by richardtorres314** [comment]
I agree, apt to start for sure! Alright, I'll add this on my list to get to; hopefully sooner than later 😄 I'll keep you posted!
